lib/rules/no-unsupported-keywords.js

```diff
@@ -16,12 +16,12 @@ function checkExpression(context, expression, allKeywords) {
   var calleeText = source.getText(expression);
   var assertionText = calleeText.substr(expectText.length + 1);
 
-  var keywords = assertionText.split(/[\.\s]/).filter(function (k) {
+  var keywords = assertionText.split(/[.\s]/).filter(function (k) {
```

Comment:
I know `.` doesn't actually need to be escaped in a `[]` set but I prefer to do it anyway for clarity and to avoid mistakes since outside of the `[]` context it would need to be escaped if the intention is to only match a `.`. So would prefer this line unchanged.

Reply:
This is part of the new eslint recommended rules: https://eslint.org/docs/rules/no-useless-escape . Do you want me to disable this rule in your config then (or inline)?

Reply:
Ha! Didn't spot that. Nah if that is what eslint is recommending, let's go with it. No point arguing with a linter... ;)

Reply:
lol :)